src/rdo.rs

```diff
@@ -223,22 +222,23 @@ fn compute_rd_cost(
       }
     };
   }
-  // Compute rate
-  let rate = (bit_cost as f64) / ((1 << OD_BITRES) as f64);
+  distortion
+}
 
+fn compute_rd(fi: &FrameInvariants, bit_cost: u32, distortion: u64) -> f64 {
```

Comment:
cost should only be used to refer to the RD cost:
```suggestion
fn compute_rd(fi: &FrameInvariants, rate: u32, distortion: u64) -> f64 {
```